xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1047 @@
+    </para>
+<screen>&prompt.user;<command>mkdir <replaceable>/tmp/skopeodata</replaceable></command></screen>
+<screen>&prompt.user;<command>helm-mirror inspect-images <replaceable>/tmp/charts</replaceable> -o skopeo --file-name <replaceable>sync.yaml</replaceable></command></screen>
+   </step>
+   <step>
+    <para>
+     Download all the referenced images using <command>skopeo</command>.
+    </para>
+<screen>&prompt.user;<command>cd /tmp/skopeodata/</command></screen>
+<screen>&prompt.user;<command>skopeo sync --source-yaml <replaceable>sync.yaml</replaceable> dir:<replaceable>/tmp/skopeodata</replaceable></command></screen>
+    <para>
+     <command>skopeo</command> will automatically create a directory named
+     after the hostname of the registry from which you are downloading the
+     images. The final path will be something like <filename>/tmp/skopeodata/registry.suse.com/</filename>.
+    </para>
+   </step>
+   <step>
+    <para>
+     Populate the local registry with the downloaded data.
+    </para>
+    <para>
+     For <literal>--dest-creds</literal> you must use the credentials you
+     created during <xref linkend="sec.deploy.scenarios.airgap.container-registry.mirror" />.
+    </para>
+<screen>&prompt.user;<command>skopeo sync --dest-creds <replaceable>USERNAME:PASSWORD</replaceable> \
+dir:<replaceable>/tmp/skopeodata/registry.suse.com/</replaceable> <replaceable>docker://myregistry.local:5000</replaceable></command>
+     </screen>
+   </step>
+   <step>
+    <para>
+     After the synchronization is done, you can remove the <filename>skopeodata</filename>
+     directory.
+    </para>
+<screen>&prompt.user;<command>rm -rf /tmp/skopeodata</command></screen>
+   </step>
+   <step>
+    <para>
+     Replace the charts download URL in the index file for the charts
+     repository and rename the file back to <filename>index.yaml</filename>.
+    </para>
+    <screen>
+&prompt.user;<command>cd /tmp/charts</command>
+</screen><screen>
+&prompt.user;<command>sed -i -e 's/https:\/\/kubernetes-charts.suse.com/https:\/\/<replaceable>myregistry.local</replaceable>\/charts/g' \
+downloaded-index.yaml</command>
+</screen><screen>
+&prompt.user;<command>mv downloaded-index.yaml index.yaml</command>
+    </screen>
+   </step>
+  </procedure>
+
+   <procedure>
+    <title>Transfer Data To Secure Storage</title>
+    <step>
+     <para>
+      Connect the trusted storage to the external mirror. We will assume the
+      device is mounted at <filename>/mnt/storage</filename>.
+     </para>
+    </step>
+    <step>
+     <para>
+      Transfer the container image data to the trusted storage. This will remove
+      all files and directories that are no longer present on the external host
+      from the trusted storage.
+     </para>
+<screen>&prompt.user;<command>rsync -av /tmp/skopeodata/registry.suse.com/ <replaceable>/mnt/storage/container-images/</replaceable> --delete</command>
```

Comment:
Maybe worth using `--progress` and dropping `-v` in this and subsequent rsync commands.